packages/cli-repl/src/cli-repl.spec.ts

```diff
@@ -799,6 +810,13 @@ describe('CliRepl', () => {
         }
       });
 
+      it('completes the db.version method', async() => {
+        output = '';
+        input.write('db.vers\u0009\u0009');
```

Comment:
Just super random thought .. we could have `const TAB = "\u0009";` and `db.vers${TAB}${TAB}` or something .. not a life changing one :)

Reply:
Sounds good, we already do that for e.g. arrow-up in the other tests. I’ll do that right after merging this.